app/assets/stylesheets/sections/app.scss

```diff
@@ -0,0 +1,39 @@
+#app-content {
+  padding-top: 75px;
+  min-height: 80%;
+}
+
+.user-logo {
+  display: block;
+  text-align: center;
```

Comment:
Properties should be ordered display, font-size, margin-left, margin-right, margin-top, text-align